core/core-impl/src/main/java/io/openshift/appdev/missioncontrol/core/impl/MissionControlImpl.java

```diff
@@ -92,16 +100,24 @@ public Boom launch(CreateProjectile projectile) throws IllegalArgumentException
         File path = projectile.getProjectLocation().toFile();
         String repositoryDescription = projectile.getGitHubRepositoryDescription();
         GitHubRepository gitHubRepository = gitHubService.createRepository(projectName, repositoryDescription);
+        event.fire(new StatusMessageEvent(projectile.getId(), StatusMessage.GITHUB_CREATE, singletonMap("location", gitHubRepository.getHomepage())));
```

Comment:
Your project will never be done.. don't know if / how we could improve that. I think error handling as a whole needs to be improved first